### PR TITLE
Fix typos with codespell

### PR DESCRIPTION
**Pull request by SuperSandro2000**

### CODE REVIEW

File: stuff/bootstrap/panels.less [hunk 1]
```diff
@@ -214,7 +214,7 @@
 }
 
 
-// Collapsable panels (aka, accordion)
```
Comment (cuu508) [inline stuff/bootstrap/panels.less:217]: I'd like to keep changes in bootstrap .less files to the minimum, so diffing and porting changes is easier.

File: templates/docs/projects_teams.html [hunk 2]
```diff
@@ -35,7 +35,7 @@ <h2>Projects and Check Limits</h2>
 <p>Account's check limit is shared by all projects owned by your account.
 For example, consider a Business account with two projects,
 "Project A" and "Project B". If A has 70 checks, then B cannot have more than
-30 checks, in order to not exceed Bussiness account's total limit of 100.</p>
```
Comment (cuu508) [inline templates/docs/projects_teams.html:38]: From some quick googling, "business" seems the correct spelling.

File: stuff/bootstrap/panels.less (hunk 1 above)
Comment (SuperSandro2000) [reply stuff/bootstrap/panels.less:217]: I tried to filter out dependencies but I missed this one.

**Comment by cuu508** [comment]
Looks good, thank you @SuperSandro2000 !

**Pull request merged**